pkg/spec/job.go

```diff
@@ -86,7 +86,7 @@ func (job *JobSpecMod) Transform() ([]runtime.Object, []string, error) {
 	return runtimeObjects, includeResources, nil
 }
 
-func (job *JobSpecMod) CreateK8sController() (*batch_v1.Job, error) {
+func (job *JobSpecMod) createKubernetesController() (*batch_v1.Job, error) {
```

Comment:
Don't forget to rename this in the tests!

Reply:
done!